### ISSUE TITLE
Build breaks: No rule to make target 'src/audio/backend/openal.cpp'

### ISSUE THREAD

**Issue by yurivict** [qTox/qTox#4433]
Build of 1.10.1 using qmake on FreeBSD 11.

```
===>  Building for qTox-1.10.1
gmake[1]: Entering directory '/usr/ports/net-im/qTox/work/qTox-1.10.1'
gmake[1]: *** No rule to make target 'src/audio/backend/openal.cpp', needed by 'openal.o'.  Stop.
gmake[1]: Leaving directory '/usr/ports/net-im/qTox/work/qTox-1.10.1'
===> Compilation failed unexpectedly.
```

**Comment by sudden6** [comment]
The problem seems to be https://github.com/qTox/qTox/blob/v1.10.1/qtox.pro#L427 and https://github.com/qTox/qTox/blob/v1.10.1/qtox.pro#L548 which shouldn't be present. Can you remove them and see if it's fixed?

Additionally, would you be interested in helping with porting the FreeBSD build to cmake?

**Comment by yurivict** [comment]
Removing openal.h and openal.cpp from .pro helps.

> Additionally, would you be interested in helping with porting the FreeBSD build to cmake?

I will look, but I can't promise. I am not too familiar with cmake. CMakeCache.txt is riddled with semicolon-separated CFLAGS and LIBS.

**Comment by sudden6** [comment]
The problem should be fixed in the v1.10.2 release.

If you could make cmake on FreeBSD work it would be great. IIRC in the other issue about that I pinpointed the problem to some GTK libs, but I'm not entirely sure.

**Comment by yurivict** [comment]
This has been fixed. Thanks!
